src/robot/parsing/suitestructure.py

```diff
@@ -49,13 +49,15 @@ def visit(self, visitor):
 class SuiteStructureBuilder(object):
     ignored_prefixes = ('_', '.')
     ignored_dirs = ('CVS',)
+    data_sources = []
```

Comment:
Hi Son,
this small shange would create a dependency between our testsuites-management and our robotframework core.
We should avoid this dependency, otherwise we can't make testsuites-management as OSS version.

This is a general issue. If we are able to run the .robot files in processes, somehow we need to distinguish on which robotframework core we run. If we run on ours, the testsuites management might need to behave different, then when it runs on the official OSS version.
Or we manage that testsuites-management runs on boths cores because the change happens transparantly in the robotframework core.

Thank you,
 Thomas

Reply:
Hello Thomas,

     I got your opinion, so I'll find out the way to modify our testsuites management behaviour instead of this change in the robotframework core.

Thank you,
Son

Reply:
Hi Son,
very good! 
I let this pull request open for the case that there is no other way to solve this problem.
Thank you,
 Thomas